course_flow/templates/course_flow/base.html

```diff
@@ -16,6 +16,22 @@
     <!-- * Fonts and icons -->
 
     <!-- * Scripts -->
+    <!-- Matomo -->
```

Comment:
If using Matomo, I think you will need to add opt-in/opt-out functionality for tracking.  Check the latest setup for Matomo in dalite (peerinst/templates/_base.html and peerinst/urls.py).   Also, does this work without the nonce?

Reply:
I really don't even remember putting this in. Looking through, this seems to have come in from my saltise-analytics branch. I must have put it in based on a suggestion from  you or Sameer, but then I paused work on that branch and now I don't remember why. It does not, in fact, run. Like you said, should have a nonce. I'm gonna just comment it out for now until I actually have a use for it.